scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -72,7 +72,7 @@ task GetToolVersions {
     # there are a handlful of tasks that require the larger GNU libc-based `slim`.
     String cloud_sdk_slim_docker = "gcr.io/google.com/cloudsdktool/cloud-sdk:435.0.0-slim"
     String variants_docker = "us.gcr.io/broad-dsde-methods/variantstore:2023-09-27-alpine"
-    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_09_13"
+    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_09_27_4a061605"
```

Comment:
Note that I have a PR https://github.com/broadinstitute/gatk/pull/8531 in which I have built and tagged `us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_09_27` with my updates, so if you need to use that tag, please rebuild / repush / retag.